src/components/DisplayPages/DisplayPageMain.js

```diff
@@ -2,6 +2,12 @@ import React from 'react'
 import DisplayPageTitle from './DisplayPageTitle'
 import styled from 'styled-components/macro'
 import VolumeSlider from '../../components/VolumeSlider'
+import PropTypes from 'prop-types'
+
+DisplayPageMain.propTypes = {
+  masterVolume: PropTypes.number.isRequired,
```

Comment:
Here I get the same warning as well: Failed prop type: Invalid prop `masterVolume` of type `string` supplied to `DisplayPageMain`, expected `number`.